src/style.css

```diff
@@ -0,0 +1,152 @@
+
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: 300;
+  padding-top: 20px ;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+input {
+  padding: 0;
+}
+
+.form1 {
+  margin-top: 0;
+  margin-bottom: 20px;
+}
+
+.form2 {
+  margin-top: 0;
+}
+
+.search-big {
+  width: 100%;
+  line-height: 22px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-top: 25px;
+  padding-left: 62px;
+  padding-bottom: 21px;
+}
+
+.search-big::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  line-height: 21px;
+}
+
+.search-big:hover {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-big:focus {
+  box-sizing: border-box;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search-big:focus::placeholder {
+  font-family: "Avenir-Heavy", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  line-height: 21px;
+}
+
+.search-icon1 {
```

Comment:
It looks like `.search-icon-small`. Or element with modifier: `.search__icon .search__icon--small` if you know about BEM methodology